projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -40,6 +41,28 @@
 				}
 			} );
 		}
+
+		var collapseButton = adminMenu.querySelector( '#collapse-button' );
```

Comment:
Check if `adminMenu` exists before operating on it. It's unlikely but this could return `null` if the element doesn't exist and the following code will fail.

Reply:
Good point! Done!